index.html

```diff
@@ -43,6 +43,138 @@ <h1>I’m Fahim Glad to see you!</h1>
           </div>
         </div>
       </section>
+      <section class="pf-works">
+        <div class="pf-portfolio">
+          <div class="card">
+            <img src="./img/tonic.png" alt="Tonic section" />
+            <h2>Tonic</h2>
+            <div class="under-header-text">
+              <p>CANOPY</p>
+              <ul>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
```

Comment:
- [ ] I would kindly suggest you add the project description to each, as this talks about what each of the projects in the card entails. Currently, the details are missing from all your cards, by implementing this you have followed the Figma template as much as possible.

Your design 👇 

![farim](https://user-images.githubusercontent.com/78634841/212252366-3191c64b-34f7-46aa-b761-2da1a5d27101.png)

Figma design 👇 

![card real](https://user-images.githubusercontent.com/78634841/212252449-b83b6f88-8c85-49c3-b337-3c63ada46ca9.png)